### ISSUE TITLE
auto-play radio shows 

### ISSUE THREAD

**Issue by kckthx** [N-Pex/nuus#12]
When selecting a radio show from the "old radioshows" view, it should start playing when selected. Currently it just populates at the top and you have to press play.

**Comment by N-Pex** [comment]
Yes, need to look into this, on iOS there's the thing that you are not allowed to "auto play" anything without the users consent.
